### DISCUSSION TITLE
Admin translation and localization

### DISCUSSION THREAD

**Discussion by techpet** [payloadcms/payload#556] [Q&A] [answered]
New to payload and already enjoying it's flexibility and customization options! Gone through most of the documentation and all localization features I've seen so far refer to content localization.

Is there a way to provide translations for the admin tokens like buttons etc.? Additionally, is there a way to provide translations for field labels in the admin panel?

**Comment by DanRibbens** [comment, marked as answer]
Hi @techpet, glad you have enjoyed Payload so far! 

We have admin panel internationalization on the roadmap. It will take us some time yet. You haven't missed anything, but we'll get there.
